### ISSUE TITLE
Gas estimation errored

### ISSUE THREAD

**Issue by williamoliver107** [HashLips/hashlips_nft_contract#44]
I can't do too many Mint at once I get the following error.
Internal JSON-RPC error. { "code": -32000, "message": "gas required exceeds allowance (29970705)" }

I don't want to have to do 200 mint at a time.

To be faster, I'd like to mint the 10k in one go.

Detail: I am using the main polygon network, and the contract is below:
0xce55d7d4a457bb07b177acb43fe5d538028bf1c8

I'm also using the account responsible for creating the smart contract.

The max mint is set to 2000 and the cost is 11 matic.

I've already tried to do the same through polygonscan and with no success.

I hope someone responds as soon as possible, that would be of great help.

Thanks.

**Comment by MarchandJD** [comment]
I'm having the same problem. Were you able to find an answer? Thanks in advance!

**Comment by williamoliver107** [comment]
> I'm having the same problem. Were you able to find an answer? Thanks in advance!

Sorry friend, I didn't find the solution so I decided to switch to the LowGas contract where I found new errors. lol

**Comment by MarchandJD** [comment]
no problem. I will just mint 100 at a time, it will cost around $50 for 8000 nfts. Minting increases the minted amount in the dapp so I'm gonna hide those numbers. All I found for batch minting is this:
https://moralis.io/how-to-bulk-mint-nfts-batch-minting-guide/
https://opensea.io/get-listed/
https://docs.opensea.io/docs/7-launching-on-mainnet
Looks like you have to insert some code for OpenSea, but it will still cost GAS. That's the name of the game I guess. Good Luck!
btw...if you want to check out my collection join my discord, send me yours too!
https://discord.gg/PKtSzGxtPZ

**Comment by williamoliver107** [comment]
> no problem. I will just mint 100 at a time, it will cost around $50 for 8000 nfts. Minting increases the minted amount in the dapp so I'm gonna hide those numbers. All I found for batch minting is this: https://moralis.io/how-to-bulk-mint-nfts-batch-minting-guide/ https://opensea.io/get-listed/ https://docs.opensea.io/docs/7-launching-on-mainnet Looks like you have to insert some code for OpenSea, but it will still cost GAS. That's the name of the game I guess. Good Luck! btw...if you want to check out my collection join my discord, send me yours too! https://discord.gg/PKtSzGxtPZ

I was able to mint 500 out of 500 using LowGas and setting the cost very low to mint.

Thank you for the link. And now I'm part of your community.

I'll be back soon to update you with my collection.

**Comment by fraroro** [comment]
hi mates, I need help: I don't know how to mint a 200 but also more (1000?) nfts at once. I'm on polygonscan and my minting price is 0.02 matic so I wrote in the contract "mint= 4" "_mintAmount (uint256)= 200" (for 200 nfts) I send to metamask and I changed just the "max fee" (the last row) on 60 gwei...but it still in pending (before I tried with 40 gwei the transaction failed in 5 minutes) so I accelerate it by writing 600 gwei but nothing, it's failed. But if you know more then me please, tell me more...

Thank you!!

**Comment by MarchandJD** [comment]
@fraroro I turned on the Advanced Gas Control in the Metamask Advanced Options. Then shoot for the Market price suggested by Metamask. If is too low it will fail or take a long time. I would do 200 at a time.
BTW, I think 0.02 MATIC is way too low for an NFT, that's like 5 cents.

**Comment by fraroro** [comment]
> @fraroro I turned on the Advanced Gas Control in the Metamask Advanced Options. Then shoot for the Market price suggested by Metamask. If is too low it will fail or take a long time. I would do 200 at a time.
> BTW, I think 0.02 MATIC is way too low for an NFT, that's like 5 cents.

I dont think so, I think people are not understanding how to do it. 200 is too much in Polygon network. Your are paying more than necessary. I minted from my smart contract 10k nfts at 0.02 each one with 35 gwei of gas fees. It worked…at the end I just pay in total 80€ for the entire collection 🙂

**Comment by zatchfx** [comment]
> > @fraroro I turned on the Advanced Gas Control in the Metamask Advanced Options. Then shoot for the Market price suggested by Metamask. If is too low it will fail or take a long time. I would do 200 at a time.
> > BTW, I think 0.02 MATIC is way too low for an NFT, that's like 5 cents.
> 
> I dont think so, I think people are not understanding how to do it. 200 is too much in Polygon network. Your are paying more than necessary. I minted from my smart contract 10k nfts at 0.02 each one with 35 gwei of gas fees. It worked…at the end I just pay in total 80€ for the entire collection 🙂

Hi, i am in the brink of deploying and minting on polygon, it would really help me if you tell me witch contract did you use, the lower gas or the normal one, and how many nfts do you mint per transaction, and if you can tell me how do you do to mint the entire collection for just 80€, thank you so much
